recipe/meta.yaml

```diff
@@ -7,46 +7,45 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/meson_python-{{ version }}.tar.gz
-  sha256: 9fcfa350f44ca80dd4f5f9c3d251725434acf9a07d9618f382e6cc4629dcbe84
-  patches:
-    - patches/0001-fix-overdependency-on-ninja.patch
+  sha256: 3d5b3e581d70a58a97b9c116a75e97a76cc4b4c7e75fa0658fc8392371e2f2c2
 
 build:
   skip: True # [py<37]
   script: {{ PYTHON }} -m pip install . -vv --no-deps
```

Comment:
Worth adding --no-build-isolation for this recipe I think.

Reply:
Thanks Anton! I'll go ahead and add it